### ISSUE TITLE
Fee payment mechanism

### ISSUE THREAD

**Issue by gitpusha** [gelatodigital/gelato-instadapp#22]
Add fee mechanism for non-ETH collateral

**Comment by Gauddel** [comment]
It seems this issue has been treated. We can close it?

**Comment by gitpusha** [comment]
I don't think we have support for different collateral types yet right? So we can leave it open. 

Current status: not needed (yet) but maybe later
